lib/text_config.dart

```diff
@@ -76,11 +76,11 @@ class TextConfig {
   ///
   ///     const TextConfig config = TextConfig(fontSize: 48.0, fontFamily: 'Awesome Font');
   ///     config.render(c, Offset(size.width - 10, size.height - 10, anchor: Anchor.bottomRight);
-  void render(Canvas canvas, String text, Position p,
+  void render(Canvas canvas, String text, Vector2 p,
       {Anchor anchor = Anchor.topLeft}) {
     final material.TextPainter tp = toTextPainter(text);
-    final Position translatedPosition =
-        anchor.translate(p, Position.fromSize(tp.size));
+    final Vector2 translatedPosition =
+        anchor.translate(p, Vector2Factory.fromSize(tp.size));
```

Comment:
tp.size.toVector2()

Reply:
So an extension for size too? Makes sense I guess.